test/powershell/Host/ConsoleHost.Tests.ps1

```diff
@@ -235,8 +235,8 @@ Describe "ConsoleHost unit tests" -tags "Feature" {
             $observed | Should -BeExactly "h-llo"
         }
 
-        It "Empty command should fail" {
-            & $powershell -noprofile -c ''
```

Comment:
What does empty command do now?  If it does nothing (basically emitting an empty string), should add a test for that.

Reply:
Agreed.

Reply:
Looks like I did rename it to "missing command", so I guess I'm good there?

I really need to get the pure-ps escaper in though.